### DISCUSSION TITLE
S3 Storage connection as a string

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [directus/directus#6390] [Feature Requests]
Hi,

Looking into the recent added PostgreSQL variable `BB_CONNECTION_STRING`, I thought, perhaps already in the pipeline but what do you think of adding the S3 bucket string as well if it is possible?

**Comment by rijkvanzanten** [comment]
Does S3 have a connection string format? Never heard of it! If it's supported by aws-sdk, there should be no problem supporting it in directus 👍🏻

**Reply by juancarlosjr97** [reply]
Sorry, it was a silly mistake with a platform that offers S3 strings from AWS, but I cannot find any S3 strings connection on the AWS site...

Close this one @rijkvanzanten and sorry...

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
